README.md

```diff
@@ -46,7 +46,7 @@ unleash.on('synchronized', () => {
   const variant = unleash.getVariant('app.ToggleY');
 });
 ```
-Be aware that the `initialize` function will configure a _global_ Unleash instance. If you call this method multiple times only the first call will setup a global Unleash instance. You may call `destroy()` to remove the globally configured instance.
+The `initialize` function will configure and creates a _global_ Unleash instance. When a global instance exists, calling this method has no effect. Call the `destroy` function to remove the globally configured instance.
```

Comment:
Nice! Thanks for accepting the suggestion. I think I made a mistake when suggesting it, so I've added a little fix here 😅 

```suggestion
The `initialize` function will configure and create a _global_ Unleash instance. When a global instance exists, calling this method has no effect. Call the `destroy` function to remove the globally configured instance.
```